argocd/resource_argocd_cluster.go

```diff
@@ -53,6 +56,7 @@ func resourceArgoCDClusterCreate(ctx context.Context, d *schema.ResourceData, me
 		return errorToDiagnostics(fmt.Sprintf("failed to list existing clusters when creating cluster %s", cluster.Server), err)
 	}
 
+	// Here we will filter ourselves on the list so that we are backward compatible for argo-cd server with version < v2.8.0 (see coment above)
```

Comment:
Do we need backward compat to versions below 2.8? That is outside of our promise, no?

Reply:
As mentioned In my review I don' think we need that compat, but since it's already coded in a way that doesn't harm we should leave it as is so far.